### DISCUSSION TITLE
ERROR: Internal log is being rate limited

### DISCUSSION THREAD

**Discussion by yingzihezhuzhu** [vectordotdev/vector#18029] [Q&A]
ERROR transform{component_kind="transform" component_id=access_logs_remap component_type=remap component_name=access_logs_remap}: 
vector::internal_events::remap: Internal log [Mapping failed with event.] is being rate limited.

ERROR sink{component_kind="sink" component_id=to_es_logs component_type=elasticsearch component_name=to_es_logs}:request{request_id=1762}: 
vector_common::internal_event::service: Internal log [Service call failed. No retries or retries exhausted.] is being rate limited.

I want to know how to solve the error?
If the error is nomal ?

**Comment by jszwedko** [comment]
Are there additional logs around the failed mapping? There typically would be.

**Reply by yingzihezhuzhu** [reply]
Are there additional logs around the failed mapping? There typically would be.   -------yes。
Thank you for reply.

**Comment by yingzihezhuzhu** [comment]
The additional logs are around the failed mapping.
There typically would be.
